Components/Core/Conditions/Categories/Spells.lua

```diff
@@ -211,6 +216,89 @@ if TMW.isRetail then
 			local VALUE = duration and start + (duration - c.Level) or huge
 		]],
 	})
+
+	ConditionCategory:RegisterCondition(2.6,  "EMPOWEREDSTAGE", {
+	    text = L["EMPOWEREDSTAGE"],
+	    tooltip = L["EMPOWEREDSTAGE_DESC"],
+	    unit="player",
+		useSUG = "spell",	
+	    name = function(editbox)
+			editbox:SetTexts(L["SPELLTOCOMP1"], L["CNDT_ONLYFIRST"])
+		end,
+	    min = 0,
+	    max = 5,
+	    icon = "Interface\\Icons\\inv_10_enchanting2_elementalswirl_color1",
+	    tcoords = CNDT.COMMON.standardtcoords,
+
+	    specificOperators = {["<="] = true, [">="] = true, ["=="]=true, ["~="]=true},
+
+	    applyDefaults = function(conditionData, conditionSettings)
+	        local op = conditionSettings.Operator
+
+	        if not conditionData.specificOperators[op] then
+	            conditionSettings.Operator = "<="
+	        end
+	    end,
+
+		events = function(ConditionObject, c)
+			return
+				ConditionObject:GenerateNormalEventString("TMW_CNDT_EMPOWERED_UPDATED")
+		end,
+		funcstr = function(c)
+			local module = CNDT:GetModule("TMWST_EMPOWERED_CAST", true)
+			if not module then
+				module = CNDT:NewModule("TMWST_EMPOWERED_CAST", "AceEvent-3.0")
+				module.Timers = {}
+				module:RegisterEvent("UNIT_SPELLCAST_EMPOWER_START", function()
+					local unit = 'player'
+					local name, text, texture, startTime, endTime, isTradeSkill, notInterruptible, spellID, _, numStages = UnitChannelInfo(unit);
+
+					if not name then return end
+
+					Env.EmpoweredStage = 0
+					Env.EmpoweredSpellName = name
+					Env.EmpoweredNumStages = numStages + 1;
+
+					local sumDuration = 0;
+
+					TMW:Fire("TMW_CNDT_EMPOWERED_UPDATED")	
+
+					local getStageDuration = function(stage)
+						if stage == Env.EmpoweredNumStages then	
+							return GetUnitEmpowerHoldAtMaxTime(unit);
+						else
+							return GetUnitEmpowerStageDuration(unit, stage-1);
+						end
+					end;
+
+					for i = 1,Env.EmpoweredNumStages-1,1 do
+						local duration = getStageDuration(i);
+
+						if(duration > 0) then
+							sumDuration = sumDuration + duration;
+							module.Timers[i] = C_Timer.NewTicker(sumDuration/1000, function()
```

Comment:
since there's no direct update event on this (`UNIT_SPELLCAST_EMPOWER_UPDATE` exists in docs but is not fired as far as I could tell), the only thing i could think of is to set a timer and then make sure to clean it up when the `STOP` event fires